packages/now-cli/src/commands/dev/dev.ts

```diff
@@ -86,5 +87,5 @@ export default async function dev(
   process.once('SIGINT', () => devServer.stop());
   process.once('SIGTERM', () => devServer.stop());
 
-  await devServer.start(...listen);
+  await devServer.start(client, link.project, ...listen);
```

Comment:
@TooTallNate The problem with this approach is now `DevServer` needs an idea of Client and the project